app/pandas.ipynb

```diff
@@ -234,18 +576,96 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "data[\"difference\"] = data[\"Age\"] - data[\"Average age\"]\n",
```

Comment:
To calculate the absolute difference, use the `abs()` function to ensure the result is non-negative. Modify the line to: `data["difference"] = (data["Age"] - data["Average age"]).abs()`.